cl/recap/tasks.py

```diff
@@ -1429,12 +1434,56 @@ def mark_fq_status(fq, msg, status):
     fq.save()
 
 
+@app.task(bind=True, max_retries=5, interval_start=5 * 60)
+def send_docket_to_webhook(d_pk: int, since: datetime, epq_pk: int) -> None:
+    d = Docket.objects.get(pk=d_pk)
+    docket_entries = DocketEntry.objects.filter(
+        date_created__gte=since, docket=d
```

Comment:
You can remove the docket query above with:

```suggestion
        date_created__gte=since, docket_id=d_pk
```